Documentation/teaching/labs/interrupts.rst

```diff
@@ -723,12 +732,12 @@ Now start the VM and insert the module:
 
 .. code-block:: shell
 
-   root@qemux86:~# skels/interrupts/kbd.ko
+   root@qemux86:~# insmod skels/interrupts/kbd.ko
    kbd: loading out-of-tree module taints kernel.
    insmod: can't insert 'skels/interrupts/kbd.ko': Device or resource busy
 
 Notice that you get an error when trying to request the I/O
-ports. This is because we already have a driver that has requestedthe
+ports. This is because we already have a driver that has requested the
 I/O ports. To validate check the /proc/ioports file for the STATUS_REG
```

Comment:
Use monospace for `/proc/ioports`, `STATUS_REG` and `DATA_REG`

Reply:
Fixed.